src/state/transactions/updater.tsx

```diff
@@ -81,17 +73,6 @@ export default function Updater() {
 
       const tx = transactions[chainId]?.[hash]
 
-      if (tx.info.type === TransactionType.SWAP && trade) {
```

Comment:
we'll have to fix how logging for analytics works for completed swaps - most likely by just adding in a serializable version of `trade` details to the `transactions` store